client/Questions&Answers/AnswersList.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+
+// eslint-disable-next-line import/extensions
+import answers from './answers.js';
+
+function Answerslist() {
+
+  return (
+    <div className="answersList">
+      {answers.map((answer) => (
+        <div key={answer.answer_id} className="answer">
+          <span>
+            <strong>A:  </strong>
+            {answer.body}
+          </span>
+        </div>
+    ))}
+    </div>
+);
+    }
+
+// Answerslist.propTypes = {
+//   answers: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.bool, PropTypes.object, PropTypes.array]).isRequired
+// }
```

Comment:
Why is this commented out? Did you disable this rule, maybe?

Reply:
Nah, I've been going back and forth about how I want to access the answers (through questions or through an api call). For now, I'm importing the dummy data directly to the page which doesn't require me to validate it as it's not a prop. I left this as a comment because I will need it once I pull actual data.